### PR TITLE
Add Kanister function ExportRDSSnapshotToLocation

### PR DESCRIPTION
**Pull request by PrasadG193**
## Change Overview
Depends on: https://github.com/kanisterio/kanister/pull/469

ExportRDSSnapshotToLocation function spins up temporary RDS instance from snapshotId creates and uploads dump to object store.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Comment by PrasadG193** [comment]
Sample blueprint:
```
apiVersion: cr.kanister.io/v1alpha1
kind: Blueprint
metadata:
  name: export-blueprint
  namespace: kasten-io
actions:
  backup:
    type: Namespace
    phases:
    - func: ExportRDSSnapshotToLocation
      name: exportSnapshot
      args:
        securityGroupID: <SGID>
        instanceID: <RDS_INSTANCE_ID>
        region: <AWS_REGION>
        username: <PGUSER>
        password: <PGPASSWORD>
        dbengine: "PostgreSQL"
        snapshotID: <SNAPSHOT_ID>
        backupArtifactPrefix: "pgbackup/rds"
```

File: pkg/function/export_rds_snapshot_location.go [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+func prepareCommand(dbEngine RDSDBEngine, action RDSAction, instanceID, dbEndpoint, username, password, backupPrefix string, profile *param.Profile) ([]string, string, error) {
+	switch dbEngine {
+	case PostgrSQLEngine:
+		switch action {
+		case BackupAction:
+			command, err := postgresBackupCommand(instanceID, dbEndpoint, username, password, backupPrefix, profile)
+			return command, postgresToolsImage, err
+		case RestoreAction:
+			fallthrough
+		default:
```
Comment (tdmanv) [inline pkg/function/export_rds_snapshot_location.go]:
```suggestion
```

File: pkg/function/export_rds_snapshot_location.go [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+		}
+	default:
```
Comment (tdmanv) [inline pkg/function/export_rds_snapshot_location.go]:
```suggestion
```

File: pkg/function/export_rds_snapshot_location.go [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+	}
+
+	tmpInstanceID := fmt.Sprintf("%s-%s", instanceID, randomID)
+	log.Print("Restore RDS instance from snapshot.", field.M{"SnapshotID": snapshotID, "InstanceID": tmpInstanceID})
+	// TODO: Use RDSRestoreSnapshot function instead
+	sgIDs, err := findSecurityGroups(ctx, rdsCli, instanceID)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to fetch security group ids. InstanceID=%s", instanceID)
+	}
+	_, err = rdsCli.RestoreDBInstanceFromDBSnapshot(ctx, tmpInstanceID, snapshotID, sgIDs)
```
Comment (tdmanv) [inline pkg/function/export_rds_snapshot_location.go]: please merge this line with the one below it

**Comment by PrasadG193** [comment]
Changes already merged as a part of https://github.com/kanisterio/kanister/pull/477. Hence closing the PR.

**Pull request closed without merging**